inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -50,6 +50,7 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue: 32032
         R"(.*ActivationParamLayerTest.*)",
         // TODO: Issue: 37862
-        R"(.*ReverseSequenceLayerTest.*netPRC=(I8|U8).*)"
+        R"(.*ReverseSequenceLayerTest.*netPRC=(I8|U8).*)",
+        R"(.*InterpolateLayerTest.*)"
```

Comment:
There are still failing tests in the pre-commit. Please, disable them.

Reply:
Tests were disabled.